onnxruntime/core/providers/cpu/rnn/uni_directional_lstm.cc

```diff
@@ -257,10 +272,15 @@ void UniDirectionalLstm<T>::Compute(const gsl::span<const T>& inputs_arg,
   const int hidden_size_x4 = 4 * hidden_size_;
   const int total_rows = max_sequence_length * batch_size_;
 
+  AllocateQuantizeBuffers<WeightT>(max_sequence_length);
+
   // apply the weights to all the inputs and save to output_IOFC
   ComputeGemm(total_rows, hidden_size_x4, input_size_, alpha, inputs.cbegin(), inputs.cend(),
               input_weights,
-              beta, output_iofc_.begin(), output_iofc_.end(), hidden_size_x4, allocator_, thread_pool_);
+              beta, output_iofc_.begin(), output_iofc_.end(), hidden_size_x4,
+              quantized_input_or_a_.begin(),
+              quantize_agg_C_.begin(),
```

Comment:
For this case, the C temp buffer is not needed, because beta=0.0 and that path is not used in ComputeGemm. Maybe just pass nullptr here to make it clear it's not used? Then you reduce the size of the C temp buffer, I think to just batch_size * hidden_size_x4.

Reply:
Nice finding. Will change it.